### PR TITLE
TRAPI 1.5

### PR DESCRIPTION
**Pull request by uhbrar**
Updates strider to be in compliance with TRAPI 1.5. Also updates URLs to no longer use TRAPI version in URL.

### CODE REVIEW

**Review by maximusunc — APPROVED** [review]
lgtm

File: tests/test_lookup.py [hunk 1]
```diff
@@ -487,11 +527,77 @@ async def test_node_set(monkeypatch, mocker):
...
+    )
+    QGRAPH = query_graph_from_string(
+        """
+        n0(( ids[] CHEBI:6801 ))
+        n0(( categories[] biolink:ChemicalSubstance ))
+        n1(( categories[] biolink:Disease ))
+        n0-- biolink:treats -->n1
+        """
+    )
+    QGRAPH["nodes"]["n1"]["set_interpretation"] = "ALL"
```
Comment (uhbrar) [inline tests/test_lookup.py:595]: @maximusunc Can we have this test use a non-set query?

**Review by uhbrar — COMMENTED** [review]
Besides the test, the changes to the caching looks good.

File: tests/test_lookup.py (hunk 1 above)
Comment (maximusunc) [reply tests/test_lookup.py]: Sure.

**Pull request merged**